### PR TITLE
feat(triggers):(DO NOT MERGE) Adding ability to skip triggering pipelines on holidays

### PR DESCRIPTION
**Pull request by jeyrschabu**
- a pipeline trigger with `skipHolidays: true` should prevent that pipeline from being triggered on a defined holiday

### CODE REVIEW

File: echo-pipelinetriggers/src/test/groovy/com/netflix/spinnaker/echo/pipelinetriggers/monitor/SkipHolidaysTriggerCriteriaSpec.groovy [hunk 1]
```diff
@@ -0,0 +1,48 @@
+/*
```
Comment (jeyrschabu) [inline echo-pipelinetriggers/src/test/groovy/com/netflix/spinnaker/echo/pipelinetriggers/monitor/SkipHolidaysTriggerCriteriaSpec.groovy]: This test is currently incomplete

**Comment by jeyrschabu** [comment]
@cfieber what do you think? Open to alternatives.

**Comment by jeyrschabu** [comment]
Supports basic & google calendar. Basic calendar can be constructed as below:
```
pipelineTriggers:
  calendar:
    timezone: America/Los_Angeles
    google:
      enabled: false
      calendarId: en.usa#holiday@group.v.calendar.google.com
      credentials: ...
    
    basic:
      enabled: true
      holidaySpecs:
        - name: Chrismas eve
          day: 24
          month: 12

        - name: Chrismas
          day: 25
          month: 12

        - name: New Year's Eve
          day: 31
          month: 12
```

**Comment by robzienert** [comment]
@jeyrschabu Like the idea. What's the fate?

**Comment by jeyrschabu** [comment]
@robzienert this is still a thing. Has been on hold for a bit but needs to be revisited

**Comment by jeyrschabu** [comment]
Going a different direction.

**Pull request closed without merging**